Frontend/ui-driver/src/Components/RideActionModal/View.purs

```diff
@@ -688,3 +688,9 @@ getCategorizedVariant variant = case variant of
       "AUTO_RICKSHAW" -> "Auto Rickshaw"
       _ -> var
   Maybe.Nothing -> ""
+
+getRideTypeColor :: Maybe.Maybe String -> String
+getRideTypeColor variant = case getCategorizedVariant variant of
+    "AC Taxi" -> Color.blue800
+    "Non AC" -> Color.orange900
+    _ -> Color.black800
```

Comment:
move this function to driver's utils

Reply:
moved the function to the driver's utils